asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryTechReport.tex

```diff
@@ -0,0 +1,119 @@
+                           & Cell Specification URL                & \url{http://vruzend.com/SpecificationINR18650MJ1.pdf}                                                                                       \\ \cline{2-3}
+                         & Cell MSDS URL                         & \url{http://bigship.com/telechargement?doc=148635}                                                                                          \\ \hline
+5.2.D.2                  & Stock Number                          & INR18650 MJ1                                                                                                                          \\ \hline
+5.2.D.3                  & Cell \& Module Voltage                & \begin{tabular}[c]{@{}l@{}}3.635 V (nominal)\\ 4.200 V (max)\end{tabular}                                                               \\ \hline
+5.2.D.4                  & Bus Voltage                           & \begin{tabular}[c]{@{}l@{}}130.86 V (nominal)\\ 151.2 V (max)\end{tabular}                                                              \\ \hline
+\multirow{3}{*}{5.2.D.5} & Cells per module                      & 36                                                                                                                                    \\ \cline{2-3}
+                         & Modules per string                    & 36                                                                                                                                    \\ \cline{2-3}
+                         & Strings in parallel                   & 1                                                                                                                                     \\ \cline{2-3}
+                         & Total cell count                      & 1296                                                                                                                                  \\ \hline
+  \multirow{3}{*}{5.2.D.6} & Capacity (kWh)                        & $\frac{3.5\text{Ah} \times 130.86\text{V} \times 36}{1000} = 16.48836$                                                                            \\ \cline{2-3}
+                         & Weight (kg)                           & \begin{tabular}[c]{@{}l@{}}0.049 kg (max)\\ 0.048 kg \times 1296 = 63.50400 kg\end{tabular}                                           \\ \cline{2-3}
+                         & Cost (USD)                            & $3.88 \times 1296 = $5028.48                                                                                                          \\ \hline
+5.2.D.7                  & Spill/damage protocols and procedures & See MSDS                                                                                                                              \\ \hline
```

Comment:
NIT: Have the word `MSDS` hyperlink to the PDF at least and maybe paraphrase what it says.

Reply:
I'll probably just include the PDFs in this document as well?